### PR TITLE
WIKI-1322 add GWT locale metadata

### PR DESCRIPTION
**Pull request by boubaker**

### CODE REVIEW

File: wiki-webui/src/main/java/org/exoplatform/wiki/webui/UIWikiPortlet.java [hunk 1]
```diff
@@ -139,6 +144,30 @@ public ResourceBundle getResourceBundle() {
...
+        LocaleConfigService localeConfigService = getApplicationComponent(LocaleConfigService.class);
+        if (localeConfigService != null && localeConfigService.getDefaultLocaleConfig() != null
+            && localeConfigService.getDefaultLocaleConfig().getLocale() != null) {
+          locale = localeConfigService.getDefaultLocaleConfig().getLocale();
+        }
+      }
+      if (locale == null) {
+        locale = Locale.ENGLISH;
+      }
+      requestMetaInformation.put("gwt:property", locale.toString());
```
Comment (thomasdelhomenie) [inline wiki-webui/src/main/java/org/exoplatform/wiki/webui/UIWikiPortlet.java]: According to your previous PR, it should be :
requestMetaInformation.put("gwt:property", "locale=" + locale.toString());

File: wiki-webui/src/main/java/org/exoplatform/wiki/webui/UIWikiPortlet.java [hunk 2]
```diff
@@ -139,6 +144,30 @@ public ResourceBundle getResourceBundle() {
...
+    }
```
Comment (thomasdelhomenie) [inline wiki-webui/src/main/java/org/exoplatform/wiki/webui/UIWikiPortlet.java]: We should put this in a dedicated method IMO, something like "setGwtLocale".

File: wiki-webui/src/main/java/org/exoplatform/wiki/webui/UIWikiPortlet.java (hunk 1 above)
Comment (boubaker) [reply wiki-webui/src/main/java/org/exoplatform/wiki/webui/UIWikiPortlet.java]: done

File: wiki-webui/src/main/java/org/exoplatform/wiki/webui/UIWikiPortlet.java (hunk 2 above)
Comment (boubaker) [reply wiki-webui/src/main/java/org/exoplatform/wiki/webui/UIWikiPortlet.java]: done

**Pull request merged**